src/Makefile

```diff
@@ -33,3 +47,7 @@ pvAccess_SYS_LIBS_WIN32 += ws2_32
 
 include $(TOP)/configure/RULES
 
+# Can't use EXPAND as generated headers must appear
+# in O.Common, but EXPAND emits rules for O.$(T_A)
+../O.Common/pvaVersionNum.h: pvaVersionNum.h@
+	$(EXPAND_TOOL) $(EXPANDFLAGS) $($@_EXPANDFLAGS) $< $@
```

Comment:
@mdavidsaver I played around with this a bit but with no joy. You don't need to set EXPANDFLAGS, that's done for you already in base/configure/RULES_EXPAND since you're using the same variable names. The rule itself should really appear in a new RULES snippet file in the src/pva sub-directory, which would need including from the bottom of the parent Makefile, but that's a minor point that can be fixed later.